backend/doc/cli.md

```diff
@@ -34,6 +34,22 @@ Useful to migrate old databases, to run before applying v3.0.0 migration scripts
 
     tracimcli db update-naming-conventions
 
+### Migrate Mysql/Mariadb database to utf8mb4 (added for tracim 3.7)
+
+:warning: This will modify the character set and collation of Tracim's database, including the already existing tables. We strongly suggest you to do a backup before applying this command.
+
+Useful to migrate old databases on mysql or mariadb in order to have full unicode support. This is mandatory starting with version 3.7 to support emoji reactions.
```

Comment:
> This is mandatory starting

What is going on if I strat tracim without apply this migration ? 
- Tracim stop with an explicit error ?
- Nothing visible but error appear when I save content ?
- error appear just if I save a reaction ?

Add information to the user.

Reply:
```suggestion
Useful to migrate old databases on mysql or mariadb in order to have full unicode support. This is mandatory starting with version 3.7 to support emoji reactions.

This fix is needed for proper working emoji character in tracim, don't applying this fix will lead to an almost working tracim, but adding emoji or complex unicode character in tracim will fail, this will make impossible to add any reaction.
```